mythos/.env.example

```
@@ -1,21 +1,27 @@
+# Buyer-side EOA address (the agent role in x402).
 BUYER_ADDRESS=
-# CDP API credentials (from https://portal.cdp.coinbase.com)
+
+# Buyer-side CDP credentials — used to sign payments AS the buyer.
+# These are NOT the backend's CDP credentials; they're agent/wallet creds for
+# the demo flow only.
 CDP_API_KEY_NAME=
 CDP_API_KEY_PRIVATE_KEY=
-# CDP wallet secret (decrypts the buyer wallet's signing key)
-CDP_WALLET_SECRET=
+CDP_WALLET_SECRET=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)